force-app/main/default/lwc/newProgramEngagement/newProgramEngagement.html

```diff
@@ -16,6 +16,16 @@
         <c-modal header={labels.newProgramEngagement} ondialogclose={handleClose}>
             <lightning-messages></lightning-messages>
             <lightning-layout multiple-rows="true">
+                <lightning-layout-item
+                    size="12"
+                    padding="around-small"
+                    if:true={allowNewContact}
```

Comment:
Is the name of this variable because we will also allow them to add a contact later?